middleware/logger/logger.go

```diff
@@ -40,7 +40,8 @@ const (
 	TagBytesReceived     = "bytesReceived"
 	TagRoute             = "route"
 	TagError             = "error"
-	TagHeader            = "header:"
+	TagReqHeader         = "reqHeader:"
```

Comment:
we should not remove the old key, otherwise we would no longer be downward compatible with the version

you can add the old tag here and in the switch, mark it as deprecated and direct it to the TagReqHeader in the switch